### PR TITLE
Accessibility statement

### PR DESCRIPTION
**Pull request by tvararu**
### Context

Add an accessibility statement to the user, based on the template in the [Service Manual](https://www.gov.uk/service-manual/helping-people-to-use-your-service/publishing-information-about-your-services-accessibility).

### Changes proposed in this pull request

Add the `/accessibility` page and link to it from the footer.

Re-engineer some of our layout code and the markdown renderer to allow us to write this page in markdown. :artist: 

### Guidance to review

image

### Checklist

https://trello.com/c/2rceoM9n/344-accessibility-page-accessibility-accessibility-statement

- [x] Attach to Trello card
- [x] Rebased master
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

File: app/views/static/accessibility.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+We’ve also made the text in the service as simple as possible to understand.
+
+<a href="https://mcmw.abilitynet.org.uk/" class="govuk-link
+govuk-link--no-visited-state">AbilityNet</a> has advice on making your device
+easier to use if you have a disability.
+
+## How accessible this service is
+
+This service is fully compliant with <a href="https://www.w3.org/TR/WCAG22/"
+class="govuk-link govuk-link--no-visited-state">the Web Content Accessibility
```
Comment (fofr) [inline app/views/static/accessibility.md:33]:
```suggestion
class="govuk-link">the Web Content Accessibility
```

File: app/views/static/accessibility.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+## Technical information about this service’s accessibility
+
+The Department for Education is committed to making this service accessible, in
+accordance with the Public Sector Bodies (Websites and Mobile Applications)
+(No. 2) Accessibility Regulations 2018.
+
+### Compliance status
+
+This service is fully compliant with <a href="https://www.w3.org/TR/WCAG22/"
+class="govuk-link govuk-link--no-visited-state">the Web Content Accessibility
```
Comment (fofr) [inline app/views/static/accessibility.md:75]:
```suggestion
class="govuk-link">the Web Content Accessibility
```

Not sure why we have `govuk-link--no-visited-state` on most of the links?

Comment (fofr) [reply app/views/static/accessibility.md:75]: (It makes sense on the email links)

**Comment by emmajhf** [comment]
Looks great @tvararu. Two teeny style things: 

1. Please can we change the last sentence in the para 'Feedback and contact information' so it reads: '...contact us if you need them in a different format, for example, large print, audio recording or braille'. The sentence that currently starts 'For example' isn't a proper sentence.
2. In 'What we are doing to improve accessibility', please can we change 'on-going' to 'ongoing'.

Thank you!

**Pull request merged**